banking-bench/src/main.rs

```diff
@@ -306,6 +315,9 @@ fn main() {
         )
         .get_matches();
 
+    let block_production_method = matches
+        .value_of_t::<BlockProductionMethod>("block_production_method")
```

Comment:
(nit: `::<BlockProductionMethod>` is redundant because rust type inference can deduce that. that said, I'd chose consistency with surrounding code. so, let's leave it for now)